doc/devguide/convert.py

```diff
@@ -0,0 +1,94 @@
+#! /usr/bin/env python
```

Comment:
We can drop this file. It exists in the userguide still, but I don't think it useful anymore.